### DISCUSSION TITLE
Deleting inside Tag

### DISCUSSION THREAD

**Discussion by Praczet** [LazyVim/LazyVim#778] [Q&A]
Hi, 

I have some issue with deleting / changing / selecting inside and outside tag for example `dit` 

So this is how it works now: 

If I want to delete inside the tag `dit`  html file being in position as on screenshot:

![Screenshot from 2023-05-19 08-56-32](https://github.com/LazyVim/LazyVim/assets/109667910/762b068b-d891-4289-a5aa-334e6cfb46ad)

it deletes everything inside `<BODY> </BODY>` but if I go go to `<svg>` tag I can delete `<circle>` and `svg` but on `<div id='time'>`  it does not work correctly. I have checked different html files and I can not find pattern for this behavior. 

I have started `nvim --clean` and it works properly. So my question is: What can cause this, and how to check it? I blocked all my plugins ... Here are my loaded plugins:

![Screenshot from 2023-05-19 09-14-16](https://github.com/LazyVim/LazyVim/assets/109667910/bd53792d-15cc-40c6-90fe-1fd966266803)

**Comment by folke** [comment]
This should be askes/reported in the mini.ai repo

**Comment by dpetka2001** [comment]
It's a limitation of `mini.ai` with nested tags that have the same name. See suggested workaround in this [issue](https://github.com/echasnovski/mini.nvim/issues/110#issuecomment-1212277863).
